lib/charms/grafana_cloud_integrator/v0/cloud_config_requirer.py

```diff
@@ -53,15 +53,9 @@ def __init__(self, charm, relation_name = DEFAULT_RELATION_NAME):
             self.framework.observe(event, self._on_relation_broken)
 
     def _on_relation_changed(self, event):
-        if not self._charm.unit.is_leader():
```

Comment:
Can you please confirm this works for both multi unit agent AND multi unit grafana cloud integrator?

Reply:
Resolving given @Abuelodelanada's review, but if anyone disagrees we can open another issue and fix it